File: src/test/java/de/gauss/aoc/ex1/Ex1Test.java
package de.gauss.aoc.ex1;

import de.gausss.aoc.ex1.Ex1;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Ex1Test {
    private Ex1 exercise = new Ex1("ex1-test.txt");

    @Test
    void testA() {
        assertEquals(exercise.solveA(), 514579);
    }

    @Test
    void testB() {
        assertEquals(exercise.solveB(), 241861950);
    }
}

File: src/test/kotlin/ex1/Ex1Test.kt
package ex1

import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test
import readInputNumbers
import kotlin.test.assertEquals

internal class Ex1Test {

    var input: List<Int> = listOf();

    @BeforeEach
    internal fun setUp() {
        input = readInputNumbers("ex1-test.txt")
    }

    @Test
    fun testA() {
        assertEquals(ex1.solveA(input), 514579)
    }

    @Test
    fun testB() {
        assertEquals(ex1.solveB(input), 241861950)
    }
}